### DISCUSSION TITLE
Flutter builds stuck in the "queued" status

### DISCUSSION THREAD

**Discussion by mkondel** [codemagic-ci-cd/codemagic-docs#2447] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Trigger a build and watch it stay in the queued status. This is the same for manual or github triggered builds. The same pipeline worked in the past.

### Expected results

A build machine gets created, it pulls code from github, builds, then pushes the bundle to google play.

### Actual results

Nothing happens, build just sits in the queued status.

### Build id (optional)

66059e2c5cb80c77e24710ca

**Comment by mikhail-tokarev** [comment, marked as answer]
please see status.codemagic.io page and this post https://github.com/orgs/codemagic-ci-cd/discussions/2424#discussioncomment-8940664
